### PR TITLE
Update install-app-guard.md

### PR DESCRIPTION
**Pull request by ekarafezov**
Adding location to Office-specific policies in group policy editor. @Dansimp @beschorr

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/install-app-guard.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ Upon being opened, the file should display a few visual indicators that the file
 ## Configure Application Guard for Office
 
 Office supports the following policies to enable you to configure the capabilities of Application Guard for Office. These policies can be configured through Group policies or through the Office cloud policy service.
+Policy settings can be found in **User Configuration\Administrative Templates\Microsoft Office 2016\Security Settings\Trust Center\Application Guard.**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/install-app-guard.md:118]:
```suggestion
Policy settings can be found in **User Configuration\Administrative Templates\Microsoft Office 2016\Security Settings\Trust Center\Application Guard**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Comment by yogkumgit** [comment]
@ekarafezov

**Comment by ekarafezov** [comment]
Yes, bold should end after the period. Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]
Good to go. I deleted my suggestion.

**Comment by ekarafezov** [comment]
Anything else need to merge this change? I'm new to this process but I anticipate more frequent updates may be needed and coming up as I hear more customer feedback on this article. Thanks!

**Comment by JohanFreelancer9** [comment]
@yogkumgit

**Comment by yogkumgit** [comment]
@maccaoz-zz Please merge this PR. Thanks

File: microsoft-365/security/office-365-security/install-app-guard.md (hunk 1 above)
Comment (tiburd) [inline microsoft-365/security/office-365-security/install-app-guard.md]:
```suggestion
Policy settings can be found at **User Configuration** > **Administrative Templates** > **Microsoft Office 2016** > **Security Settings** > **Trust Center** > **Application Guard**.
```

**Comment by tiburd** [comment]
@ekarafezov 

Can you please confirm that your proposed changes are still valid? Where you proposed your change, a link has been added:

- Here's the article: https://docs.microsoft.com/microsoft-365/security/office-365-security/install-app-guard
- The relevant information has been replaced with a link to this article: https://docs.microsoft.com/DeployOffice/overview-office-cloud-policy-service

Please @ mention me in your reply. Thank you!

**Comment by ekarafezov** [comment]
You can abandon this PR, it's been a while and there were later changes so this is no longer applicable.

**Pull request closed without merging**